tests/performance/compare-results.js

```diff
@@ -96,6 +96,8 @@ if ( process.env.GITHUB_SHA ) {
 	);
 }
 
+summaryMarkdown += `<details><summary>Results</summary>`;
```

Comment:
What's the rationale for these markup changes? What does this do?

Reply:
It makes the GitHub Actions workflow job summaries more readable by wrapping the results in a `<details>` tag:

<img width="972" alt="Screenshot 2024-12-19 at 11 09 43" src="https://github.com/user-attachments/assets/ae014cb0-20c8-4bb2-9ab9-66985024bf38" />